src/Adapter/GeneralConfiguration.php

```diff
@@ -98,15 +116,23 @@ public function updateConfiguration(array $configuration)
      */
     public function validateConfiguration(array $configuration)
     {
-        return isset(
-            $configuration['check_modules_update'],
-            $configuration['check_ip_address'],
-            $configuration['front_cookie_lifetime'],
-            $configuration['back_cookie_lifetime']
-        ) && in_array(
-            $configuration['cookie_samesite'],
-            Cookie::SAMESITE_AVAILABLE_VALUES
-        );
+        $isValid = isset(
+                $configuration['check_modules_update'],
+                $configuration['check_ip_address'],
+                $configuration['front_cookie_lifetime'],
+                $configuration['back_cookie_lifetime']
+            ) && in_array(
+                $configuration['cookie_samesite'],
+                Cookie::SAMESITE_AVAILABLE_VALUES
+            );
+        if ($this->isDebug) {
+            $isValid &= in_array(
+                $configuration['check_modules_stability_channel'],
+                Tools::ADDONS_API_MODULE_CHANNELS
+            );
+        }
+
+        return $isValid;
```

Comment:
```suggestion
        return (bool) $isValid;
```
because &= returns an integer

Reply:
Thanks & Done